example/ssd/dataset/iterator.py

```diff
@@ -159,21 +156,22 @@ def _data_augmentation(self, data, label):
                 xmax = int(crop[2] * width)
                 ymax = int(crop[3] * height)
                 if xmin >= 0 and ymin >= 0 and xmax <= width and ymax <= height:
-                    data = data[ymin:ymax, xmin:xmax, :]
+                    data = mx.img.fixed_crop(data, xmin, ymin, xmax-xmin, ymax-ymin)
                 else:
                     # padding mode
                     new_width = xmax - xmin
                     new_height = ymax - ymin
                     offset_x = 0 - xmin
                     offset_y = 0 - ymin
-                    data_bak = data
+                    #TODO need ndarray to support this
+                    data_bak = data.asnumpy()
```

Comment:
@piiswrong can u explain a bit more how to use layer as operator? since layer is operating against Tensor not NDArray. if I use symbolic API, what's the execute engine then?